app/services/image_upload_presigned_url_generator.rb

```diff
@@ -0,0 +1,16 @@
+class ImageUploadPresignedUrlGenerator
```

Comment:
This was just de-namespaced (formerly `DocAuth::ImageUpload...`), since it definitely belongs outside the gem